### PR TITLE
Bug 1794118: Removed the misleading warning

### PR DESCRIPTION
**Pull request by gnehapk**
Removed the warning and validations for memory and cpu check from UI.

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx [hunk 1]
```diff
@@ -35,7 +35,8 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
           isInline
         />
         <p className="co-legend co-required ceph-ocs-desc__legend">
-          Select at least 3 nodes in different failure domains you wish to use.
+          Select at least 3 nodes in different failure domains with a minimum requirements of 16
+          cores and 64 GiB of RAM per node.
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx]:
```suggestion
          CPU cores and 64 GiB of RAM per node.
```

Comment (gnehapk) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx]: @cloudbehl This message is as per the @yuvalgalanti suggestion. Also `CPU` keyword was not present even in the earlier warning message.

**Comment by gnehapk** [comment]
@cloudbehl Updated the PR. Please review.

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx [hunk 2]
```diff
@@ -35,7 +35,8 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
           isInline
         />
         <p className="co-legend co-required ceph-ocs-desc__legend">
-          Select at least 3 nodes in different failure domains you wish to use.
+          Select at least 3 nodes in different failure domains with a minimum requirements of 16
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx]:
```suggestion
          Select at least 3 nodes in different failure domains with a minimum requirement of 16
```

Comment (gnehapk) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx]: It should be `requirements` only as its referring to 2 requirements  i.e. CPU and memory

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx]: Then we should remove `a`

**Pull request merged**